airsonic-main/src/main/java/org/airsonic/player/domain/InternetRadioSource.java

```diff
@@ -0,0 +1,11 @@
+package org.airsonic.player.domain;
+
+public class InternetRadioSource {
```

Comment:
Is it really worth having a class to wrap a String?

Reply:
Maybe that's just personal preference, but what the code does sounds clearer to me when it's handling a "more qualified" type like `List<InternetRadioSource>` instead of a regular `List<String>`. Also, it's not a particularly performance-sensitive section, otherwise I'd agree with you.

Reply:
Too bad there isn't a nice way to do type aliasing in Java :/